src/main/java/com/mrh0/createaddition/energy/WireType.java

```diff
@@ -67,4 +68,18 @@ public int getBlue() {
 	public boolean isFestive( ) {
 		return this == FESTIVE;
 	}
+
+	public static WireType of(Item item) {
+		if(item == CAItems.COPPER_SPOOL.get())
+			return WireType.COPPER;
+		if(item == CAItems.GOLD_SPOOL.get())
+			return WireType.GOLD;
+		if(item == CAItems.FESTIVE_SPOOL.get())
+			return WireType.FESTIVE;
+		//if(item == CAItems.IRON_SPOOL.get())
+		//	return WireType.IRON;
+		//if(item == CAItems.ELECTRUM_SPOOL.get())
```

Comment:
Why didnt you add electrum? Did you not pull from Forge?

Reply:
No I didn't. Does that even work with so many different changes?

Reply:
I tried and there are about 1000 conflicts. I don't know if pulling from Forge is realistic...

Reply:
If you wanted to do that, you'd better use architectury, I think.

Reply:
Never mind, I can do that. ~~(However, it would still save a lot of work)~~